app/Providers/RepositoryServiceProvider.php

```diff
@@ -7,20 +7,23 @@
 use HMS\Entities\User;
```

Comment:
given the standard pattern and we should be able to do something with a config file array and a foreach loop, we can build all the full class strings with out needing to import each one, 
Think I'll start a separate issue to cover it.